pkg/tfbridge/diff_test.go

```diff
@@ -143,6 +144,47 @@ func TestCustomizeDiff(t *testing.T) {
 		assert.Equal(t, changes, pulumirpc.DiffResponse_DIFF_NONE)
 		assert.Equal(t, expectedDiff, diff)
 	})
+
+	t.Run("CustomDiffDoesNotPanicOnGetRawState", func(t *testing.T) {
+		for _, diffStrat := range []shimv2.DiffStrategy{shimv2.PlanState, shimv2.ClassicDiff} {
+			diffStrat := diffStrat
```

Comment:
go n00b question: What's the purpose of this line? I would have expected it to complain about redeclaring the variable. Is the scope of the diffStrat on the LHS different than the scope of diffStrat on the RHS?

Reply:
Without this line diffStrat will be captured by reference instead of by-value in the closure, and if the tests run in parallel they will all exercise against the  shimv2.ClassicDiff (last value in the slice).